Module_8/Q1.js

```diff
@@ -0,0 +1,13 @@
+//  Write a function that can stop execution of a function for the number of milliseconds sent as an argument
+const func = async()=>{
+  console.log("Before Printing")
+  await sleep(3000)
+  console.log("After Printing")
+}
+
+const sleep = async(time)=>{
+  let p=new Promise(resolve=> setTimeout(resolve,time))
+  return p
+}
```

Comment:
Whenever we have a function which just returns something on first line itself we can follow a shorter code syntax
`const sleep = async(time)=>new Promise(resolve=> setTimeout(resolve,time))`